docusaurus/docs/security/security-advanced-configuration.md

```diff
@@ -734,4 +768,9 @@ KEYCLOAK_CLIENT_ID
 KEYCLOAK_SECURITY_CLIENT_ID
 
 - **_Description_**: Keycloak OpenID client ID
-- **_Default_**: `"6c3465b1-2674-4704-a940-c41194dbd95"`
\ No newline at end of file
+- **_Default_**: `"6c3465b1-2674-4704-a940-c41194dbd95"`
+
+EVENT_DISPATCHER_CALLBACK_URL
```

Comment:
I'm thinking if we wouldn't need to be able to have more than one callback URL somehow

Reply:
Done